### PR TITLE
Add requirements.txt

### PR DESCRIPTION
**Pull request by Binyang2014**

### CODE REVIEW

File: .github/workflows/continuous-integration.yml [hunk 1]
```diff
@@ -28,7 +28,8 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install kubernetes pyyaml requests jinja2 pystache pylint responses
+        pip install -r requirements.txt
+        pip install -r requirements_dev.txt
```
Comment (abuccts) [inline .github/workflows/continuous-integration.yml:32]: it's better to use `extras_require` in `setup.py` if you want multiple envs

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,5 @@
+kubernetes
+pyyaml
+requests
+jinja2
+pystache
```
Comment (abuccts) [inline requirements.txt:5]: why don't freeze version

Comment (Binyang2014) [reply requirements.txt]: Version fixed

File: .github/workflows/continuous-integration.yml (hunk 1 above)
Comment (Binyang2014) [reply .github/workflows/continuous-integration.yml:32]: Since we don't want to distribute this as a python package, keep using `requirements.txt`

**Review by abuccts — APPROVED** [review]

**Pull request merged**